check_api/src/main/java/com/google/errorprone/matchers/Description.java

```diff
@@ -164,6 +180,7 @@ private Builder(
       this.linkUrl = linkUrl;
       this.severity = Preconditions.checkNotNull(severity);
       this.rawMessage = Preconditions.checkNotNull(rawMessage);
+      this.metadata = new HashMap<>();
```

Comment:
nit: `LinkedHashMap`? I love maintaining the order 😄

Reply:
Haha sure I can change that